conf/janus.plugin.streaming.jcfg.sample.in

```diff
@@ -83,6 +83,16 @@
 # assuming the browser supports the RTP extension in the first place.
 # playoutdelay_ext = true
 #
+# To allow mountpoints to negotiate the abs-capture-time RTP extension,
+# you can set the 'abscapturetime_ext' property to true: this way, any
+# subscriber can receive the abs-capture-time of incoming RTP streams,
+# assuming the browser supports the RTP extension in the first place.
+# abscapturetime_ext = true
+#
+# Extension header id in which RTP source provides abs-capture-time.
+# Should be in range 1..14 inclusive.
+# abscapturetime_src_ext_id = 1
```

Comment:
Thinking about it, I think you can get rid of the `abscapturetime_ext` boolean: in fact, since the ID can't be 0, it being non-0 implicitly sets the feature to true, thus simplifying the configuration (and the code).

Reply:
Yes, completely agree. That's exactly what I've proposed before.